README.md

```diff
@@ -17,17 +29,65 @@ Take a look at the `ServiceSpec`, in short:
         service.perform(call) { (result: Result<Posts>) in
             DispatchQueue.main.async {
                 switch result {
-                case .model(let model):
+                case .models(let model):
                     print("🎉 \(model)")
                 default:
                     print("💣 fail")
                 }
             }
         })
 ```
+## Parsing results
+
+Parsing and Serialization can happen automagically. Best is to take a look at `ParseableSpec`. A typical `Parseable` Type looks like:
+
+### Type without relations
+
+```swift
+class Foo: Parseable {
+  var uuid: String?
+  var blue: String?
 
-## Write unit tests
-In the example project you can find examples written with `Nimble` for both CoreData and others.
+  required init?(from raw: Any) {
+      map(from: raw)
+  }
+
+  var mappers: [String : ((Any?)->())] {
+      return ["uuid" : {self.uuid <- $0 },
+              "blue" : {self.blue <- $0 }]
+  }
+
+}
+```
+
+### Type with relations
+```swift
+class Foo: Parseable {
+  var uuid: String?
+  var blue: String?
+  var fooRelation: FooRelation?
+  var relations: [FooRelation]?
+
+  required init?(from raw: Any) {
+      map(from: raw)
+  }
+
+  var mappers: [String : ((Any?)->())] {
+      return ["uuid" : {self.uuid <- $0 },
+              "blue" : {self.blue <- $0 },
+              "fooRelation": {self.fooRelation = FooRelation(from: $0)},
+              "relations": addRelations()
+              ]
+  }
+
+  private func addRelations() -> (Any?)->() {
```

Comment:
relationsMappingFunction is het geworden want relations() gaat niet want is al in gebruik voor de property.